src/staveconnector.ts

```diff
@@ -255,9 +233,11 @@ export class StaveConnector extends Element {
         topY -= 6;
         botY += 6;
         attachmentHeight = botY - topY;
-        Glyph.renderGlyph(ctx, topX - 5, topY, 40, 'bracketTop');
-        Glyph.renderGlyph(ctx, topX - 5, botY, 40, 'bracketBottom');
-        topX -= this.width + 2;
+        element.setText('\ue003' /* bracketTop */);
+        element.renderText(ctx, topX - 5, topY);
+        element.setText('\ue004' /* bracketBottom */);
+        element.renderText(ctx, topX - 5, botY);
+        topX -= 5;
```

Comment:
Can we explain (via comments) what these magic numbers -5 do? Also, why do we need to modify topX to shift it to the left by 5?

Reply:
I do not know. I did not look into that. There was two magic numbers 3 and 2 and I just replaced by 5.